### PR TITLE
Npm Modules Updated and .gitignore

### LINKED ISSUE #16: Update NPM and .gitignore
We have to update .gitignore for config.js for the token.

### PR DESCRIPTION
**Pull request by aggmoulik**
Updated NPM Packages and .gitignore for a config.js file.
Resolve #16

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 node_modules/
 ./package-lock.json
 .idea/
+./config.js
```
Comment (MatejMecka) [inline .gitignore:4]: Maybe we should create a template? like config.js.template? and ignore the other one

Comment (aggmoulik) [reply .gitignore:4]: I can create a script for creating a config.js file.
Resolve this pull request I will make another pull request for the script and all other errors.

Comment (MatejMecka) [reply .gitignore:4]: Yes do that!

Comment (aggmoulik) [reply .gitignore:4]: So I have to make a PR until it is resolved.

Comment (MatejMecka) [reply .gitignore:4]: I'll merge it :) But better send that script :))

**Pull request merged**